File: Foundation/Arrays/inverse.cpp
#include<iostream>
#include<vector>
using namespace std;

int inverse(int[] arr)
{
    int n = arr.size();
    vector<int> ans(n);
    for(int i = 1; i < n; i++)
    {
        ans[arr[i]] = i;
    }
    for(int i = 0; i < n; i++)
    {
        cout << ans[i] << endl;
    }
}

int main()
{
    int n;
    cin >> n;
    vector<int> arr(n);
    for(int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }
    inverse(arr);
}

File: Foundation/Arrays/inverse.java
import java.util.Scanner;
public class inverse
{
    public static Scanner scn = new Scanner(System.in);
    public static int inverse(int[] arr)
    {
        int n = arr.length;
        int[] ans = new int[n];
        for(int i = 1; i < n; i++)
        {
            ans[arr[i]] = i;
        }
        for(int i = 0; i < n; i++)
        {
            System.out.println(ans[i]);
        }
    }

    public static void main()
    {
        int n = scn.nextInt();
        int[] arr = new int[n];
        for(int i = 0; i < n; i++)
        {
            arr[i] = scn.nextInt();
        }
        inverse(arr);
    }
}